lib/game.rb

```diff
@@ -0,0 +1,25 @@
+class Game
+  attr_reader :game_id,
+              :season,
+              :type,
+              :date_time,
+              :away_team_id,
+              :home_team_id,
+              :away_goals,
+              :home_goals,
+              :venue,
+              :venue_link
+
+  def initialize(info)
+    @game_id      = info[:game_id].to_i
+    @season       = info[:season].to_i
+    @type         = info[:type]
+    @date_time    = info[:date_time]
+    @away_team_id = info[:away_team_id].to_i
+    @home_team_id = info[:home_team_id].to_i
+    @away_goals   = info[:away_goals].to_i
+    @home_goals   = info[:home_goals].to_i
+    @venue        = info[:venue]
+    @venue_link   = info[:venue_link]
+  end
```

Comment:
As a group, are we going to right align our attributes or align them around the "="?